client/reader/conversations/intro.jsx

```diff
@@ -0,0 +1,135 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import QueryPreferences from 'components/data/query-preferences';
+import { savePreference } from 'state/preferences/actions';
+import { getPreference } from 'state/preferences/selectors';
+import { recordTrack } from 'reader/stats';
+
+const getPreferenceName = isInternal =>
+	isInternal ? 'has_used_reader_conversations_a8c' : 'has_used_reader_conversations';
+
+class ConversationsIntro extends React.Component {
+	static propTypes = {
+		isInternal: PropTypes.bool,
+	};
+
+	static defaultProps = {
+		isInternal: false,
+	};
+
+	componentDidMount() {
+		this.recordRenderTrack();
+	}
+
+	componentWillReceiveProps( nextProps ) {
+		if ( this.props.hasUsedConversations !== nextProps.hasUsedConversations ) {
+			this.recordRenderTrack( nextProps );
+		}
+	}
+
+	recordRenderTrack = ( props = this.props ) => {
```

Comment:
the name of this function is a bit deceiving. 
Its really `maybeRecordRenderTrack` 🤷‍♂️ 

FFTI, its super nitpicky